src/main/webapp/resources/js/pages/analysis/components/settings/AnalysisReferenceFileRenderer.jsx

```diff
@@ -45,6 +45,7 @@ export function AnalysisReferenceFileRenderer() {
                 onClick={() => {
                   downloadReferenceFile();
                 }}
+                className="t-reference-file-download-btn"
               >
                 <Icon type="download" />{" "}
```

Comment:
Don't like seeing `{" "}` it is generated by the editor.  If you really want spacing add it with an inline style like `marginRight: SPACED_XS`

Reply:
Updated in b7f397facd7f75e5f8d51ba00461f3f813291d2f